pkg/services/services.go

```diff
@@ -46,11 +46,33 @@ func GenerateManifest(kubeletArgsFromIgnition map[string]string, vxlanPort strin
 		hybridOverlayConfiguration(vxlanPort, debug),
 		kubeProxyConfiguration(debug),
 	}
+	if platform == config.AzurePlatformType && ccmEnabled {
+		*services = append(*services, azureCloudNodeManagerConfiguration())
+	}
 	// TODO: All payload filenames and checksums must be added here https://issues.redhat.com/browse/WINC-847
 	files := &[]servicescm.FileInfo{}
 	return servicescm.NewData(services, files)
 }
 
+// azureCloudNodeManagerConfiguration returns the service specification for azure-cloud-node-manager.exe
+func azureCloudNodeManagerConfiguration() servicescm.Service {
+	cmd := fmt.Sprintf("%s --windows-service --node-name=NODE_NAME --wait-routes=false --kubeconfig=%s",
+		windows.AzureCloudNodeManagerPath, windows.KubeconfigPath)
+
+	return servicescm.Service{
+		Name:    windows.AzureCloudNodeManagerServiceName,
+		Command: cmd,
+		NodeVariablesInCommand: []servicescm.NodeCmdArg{{
+			Name:               "NODE_NAME",
+			NodeObjectJsonPath: "{.metadata.name}",
+		}},
+		PowershellVariablesInCommand: nil,
+		Dependencies:                 nil,
+		Bootstrap:                    false,
+		Priority:                     0,
```

Comment:
kubelet has priority 0 already. The enhancement specifies each service that has the bootstrap flag set as true must have a higher priority than all non-bootstrap services.